src/resources/i18n/exchange/Messages_fr_FR.xml

```diff
@@ -61,24 +61,27 @@
-   <translation id="exchange_office_attached_alert_modale_header" qtlid="404252">Obtenez un mois gratuit d'Office 365 Business sur un maximum de {0} comptes Exchange de votre choix</translation>
-   <translation id="exchange_office_attached_alert_modale_selected_accounts" qtlid="404265">{0} comptes sélectionnés pour un maximum de {1}</translation>
-   <translation id="exchange_office_attached_alert_modale_account_types" qtlid="404278">Types de comptes à utiliser dans les recherches</translation>
-   <translation id="exchange_office_attached_alert_modale_search_form" qtlid="404291">Rechercher des comptes emails à utiliser</translation>
-   <translation id="exchange_office_attached_alert_modale_confirmation_header" qtlid="404304">Vous allez activer l'offre Office 365 sur les {0} comptes suivants :</translation>
-   <translation id="exchange_office_attached_alert_modale_confirmation_checkbox" qtlid="404317">En cochant cette case, je confirme vouloir souscrire à l'offre pour les comptes ci-dessus et je confirme avoir compris que l'offre sera reconduite automatiquement si je ne l'annule pas.</translation>
+<!-- OFFICE - One month free promotion -->
+   <translation id="exchange_office_1M_promotion_button_title">Essai gratuit d'un mois Office 365</translation>
+   <translation id="exchange_office_1M_promotion_button_title_extra">Offre promotionnelle</translation>
+   <translation id="exchange_office_1M_promotion_main_title">Activer l'offre promotionnelle Office 365</translation>
+   <translation id="exchange_office_1M_promotion_intro">Obtenez un mois gratuit d'Office 365  sur un maximum de {0} comptes Exchange.</translation>
+   <translation id="exchange_office_1M_promotion_intro_table">Sélectionnez les comptes emails de votre choix</translation>
+   <translation id="exchange_office_1M_promotion_selected_accounts">{0} comptes sélectionnés pour un maximum de {1}</translation>
+   <translation id="exchange_office_1M_promotion_search_form">Rechercher des comptes emails à utiliser</translation>
+   <translation id="exchange_office_1M_promotion_account_types">Types de comptes à utiliser dans les recherches</translation>
+   <translation id="exchange_office_1M_promotion_confirmation_table">Confirmez les comptes e-mail</translation>
+   <translation id="exchange_office_1M_promotion_confirmation_header"> {0} sur {1} comptes e-mail sélectionnés</translation>
+   <translation id="exchange_office_1M_promotion_selection_button_caption">Sélectionner</translation>
+   <translation id="exchange_office_1M_promotion_selection_change_link">Modifier mes choix</translation>
+   <translation id="exchange_office_1M_promotion_legal_conditions_caption">En cochant cette case, je confirme vouloir souscrire à l'offre pour les comptes ci-dessus et je confirme avoir compris que l'offre sera reconduite automatiquement si je ne l'annule pas. (Vous aurez un rappel X jours avant la fin de l'offre promotionnelle.)</translation> 
```

Comment:
Were these messages confirmed by Content ?

Reply:
In progress, it takes time...